cloudsql/postgres/database-sql/connect_connector.go

```diff
@@ -61,9 +57,6 @@ func connectWithConnector() (*sql.DB, error) {
 	}
 	// [START cloud_sql_postgres_databasesql_auto_iam_authn]
 	var opts []cloudsqlconn.Option
-	if dbIAMUser != "" {
```

Comment:
One small nit: we should probably leave this in. Cut the docs over. And then remove this.

Reply:
Removed the region tag from this file as it is currently not being rendered within the https://cloud.google.com/sql/docs/postgres/iam-logins page due to a bug showing the Java sample for Go.